sections/1-introduction.tex

```diff
@@ -1,101 +1,76 @@
-The project was formally founded in March of 2014 though development of the core package began three years earlier. 
-\citet{Community:2015cy} describes version 0.5 which was released in June 2014.
+Software packages to analyze data from these instruments are generally developed independently by each instrument team. 
+This creates a diverse and therefore difficult data and software environment to navigate. 
+Scientists conducting interdisciplinary research from multiple instruments run into problems with incompatible data formats and incompatible analysis routines written with different versions of the base code. Further, it is difficult to reproduce others' scientific results without open data and version-controlled open source software.
 
-The \sunpyproj selected \python to leverage the rich ecosystem of packages available for general data analysis. 
-These include packages such as \package{Numpy} which provides efficient multi-dimensional numerical array manipulation \citep{numpy} ; \package{scipy} which provides fundamental scientific functions such as for numerical integration and optimization\citep{scipy}; \package{Matplotlib} provides publication-ready 2D plotting \citep{matplotlib}, and finally \package{Pandas} provides data structures and analysis support with support for time series \citep{pandas}.
-These packages form the foundation of the scientific \python ecosystem. 
-Together they represent $>$500,000 lines of code\footnote{As measured by cloc (\url{github.com/AlDanial/cloc})}. 
-Significant additional packages have been developed which depend on these foundational packages. 
-Of particular relevance to \sunpypkg is the \astropypkg package which provides core functionality for astronomy \citep{astropy2018}. 
+A common, version-controlled, open source platform that provides a standard interface to data products and encourages the re-use of common functions can go a long way toward solving this problem. 
+SunPy aims to provide this solution for the field of solar physics.
 
-%The project began as a community-led effort to organize and standardize existing functionality and also to provide a \todo{Would "open source" be better than "free" because SSW is "free" but it's not "open source" - SJM} free and modern alternative to the existing SolarSoft (SSW, \citet{Freeland:1998we}) software package. While SSW is open source and freely available, it is primarily composed of source code for the Interactive Data Language (IDL), a proprietary data-analysis environment currently owned by Harris Geospatial Solutions. In addition, the development of SSW is not open to the community and is not version controlled.
+The mission statement of the \sunpyproj is to facilitate and promote the use and development of several community-led, free, and open-source\footnote{\url{https://opensource.org/osd}} data analysis software packages based on the scientific \python\footnote{\url{https://www.python.org/}} environment.
```

Comment:
This re-wording is now not technically our mission statement.

Reply:
I copied the italicized text from the [SunPy Project](https://sunpy.org/team) page -- is that incorrect?